.github/workflows/deploy.yaml

```diff
@@ -45,18 +45,18 @@ jobs:
         env:
           APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
           CLUSTER: dev-fss
-          RESOURCE: nais.yaml
+          RESOURCE: .nais/nais.yaml
           IMAGE: ${{ env.IMAGE }}
           TEAM: pensjonsamhandling
-          VARS: nais-dev-vars.yaml
+          VARS: .nais/nais-dev-vars.yaml
 
       # Deploy to prod-fss
-      - name: Deploy to prod-fss
-        uses: nais/deploy/actions/deploy@v1
-        env:
-          APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
-          CLUSTER: prod-fss
-          RESOURCE: nais.yaml
-          IMAGE: ${{ env.IMAGE }}
-          TEAM: pensjonsamhandling
-          VARS: nais-prod-vars.yaml
+#      - name: Deploy to prod-fss
```

Comment:
hvordan deployer til prod da?

Reply:
Deployer til dev først for å verifisere at endringene ikke ødelegger noe.